### PR TITLE
Start circle management

### LINKED ISSUE #109: As a user I want to create circles, so that I can group people
- create database (separate PR)
- form to create empty circle with the given name, located on circle list page
- redirect to edit page

### LINKED ISSUE #112: As a user I want to view all my circles and the people in it
- overview page of all own circles
- display users in each circle (at least some)
- links to edit, delete, ... each circle

### PR DESCRIPTION
**Pull request by Gjum**
Listing and creation work.
Editing is limited to circle name currently, see #111.

Closes #109, closes #112.

### CODE REVIEW

**Comment by Gjum** [comment]
@MetalMatze ping

File: circles/migrations/0002_auto_20150617_2034.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    dependencies = [
+        ('circles', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='circle',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True, default=datetime.datetime(2015, 6, 17, 18, 33, 58, 802301, tzinfo=utc)),
```
Comment (metalmatze) [inline circles/migrations/0002_auto_20150617_2034.py:19]: Why did you hardcore the time as default? Isn't something like `auto_now=True` better?

Comment (Gjum) [inline circles/migrations/0002_auto_20150617_2034.py:19]: These are the times that get written into the database when you migrate. When circles get created/updated, they get the current time.

Comment (metalmatze) [inline circles/migrations/0002_auto_20150617_2034.py:19]: You're absolutly right...My bad.

**Pull request merged**